src/lightbox/Lightbox.js

```diff
@@ -69,11 +51,40 @@ export default function Lightbox(props: Props): Node {
       ? `Shared in #${streamNameOfStreamMessage(message)}`
       : 'Shared with you';
 
-  // Since we're using `Dimensions.get` (below), we'll want a rerender
-  // when the orientation changes. No need to store the value.
-  useGlobalSelector(state => getGlobalSession(state).orientation);
+  const overlayStyles = React.useMemo(
+    () => ({
+      backgroundColor: 'black',
+      opacity: 0.8,
+      position: 'absolute',
+      width: '100%',
```

Comment:
Though actually, looking at these:
* The width is probably clearest if right by the height, so duplicated on the two of them.
* The position ditto -- together those three describe the layout, and can't really be understood without each other.
* The color and opacity OTOH are appropriate to deduplicate out here.

But for two properties, that's a pretty small payoff for the extra machinery involved in this deduplication.

In fact even for four of them, I think it'd be net fewer lines to just duplicate them all at the two sites. Probably also net clearer, even apart from the position/width/height issue.